app/views/forms/incident-report/_request-info.html.erb

```diff
@@ -0,0 +1,28 @@
+<fieldset class="webform-component-fieldset form-wrapper" id="webform-component-primary-relevant-person"><legend><span class="fieldset-legend">Primary Relevant Person, if needed</span></legend><div class="fieldset-wrapper"><div class="form-item webform-component webform-component-textfield" id="webform-component-primary-relevant-person--identify-relevant-persons">
+<%= f.input :primary_relevant_person, label: "Primary Relevant Person", as: :text, required: :true %>
+<%= f.input :primary_email, label: "Provide Email, if needed", as: :text, required: :false %>
+<%= f.input :primary_affiliation, label: "Affiliation", as: :radio_buttons, collection: ["TU Student", "TU Faculty or Staff", "Library Student Assistant", "Library Staff", "Visitor/Guest"], required: :true %>
+<fieldset class="webform-component-fieldset form-wrapper" id="webform-component-secondary-relevant-person"><legend><span class="fieldset-legend">Secondary Relevant Person, if needed</span></legend><div class="fieldset-wrapper"><div class="form-item webform-component webform-component-textfield" id="webform-component-secondary-relevant-person--identify-relevant-persons">
+<%= f.input :secondary_relevant_person, label: "Secondary Relevant Person", as: :text, required: :true %>
+<%= f.input :secondary_email, label: "Provide Email, if needed", as: :text, required: :false %>
+<%= f.input :secondary_affiliation, label: "Affiliation", as: :radio_buttons, collection: ["TU Student", "TU Faculty or Staff", "Library Student Assistant", "Library Staff", "Visitor/Guest"], required: :true %>
+<%= f.input :incident_description, label: "Incident Description", as: :text, required: :true, cols: 60, rows: 5 %>
+<div class="description">What happened?  Provide names, who did what to whom, what was stolen, etc. Be as detailed as possible. Describe a suspect if appropriate.</div>
+<%= f.input :victim_complaint_statement, label: "Victim/Complainant Statement", as: :text, required: :true, cols: 60, rows: 5 %>
+<%= f.input :action_taken, label: "Action Taken", as: :check_boxes, collection: ["Library Administration Notified", "Facilities Notified (1-1385)", "Completed Staff Injury Report", "Referred to Legal or Risk Management", "Called Campus Police"], required: :true %>
+<div class="description">Check all that apply. Provide police report # if available.</div>
+<%= f.input :police_report_number, label: "Police Report Number", as: :text, required: :true, cols: 60, rows: 128 %>
+<%= f.input :other_action_taken, label: "Other Action Taken", as: :text, required: :true, cols: 60, rows: 128 %>
+<%= f.input :supervisor_notified, as: :radio_buttons, collection: ["Yes", "No"], required: :true %></div>
+
+<input type="hidden" name="details[sid]">
+<input type="hidden" name="details[page_num]" value="1">
+<input type="hidden" name="details[page_count]" value="1">
+<input type="hidden" name="details[finished]" value="0">
+<input type="hidden" name="form_build_id" value="form-HcgpTB4ZuQCwzQ0xXM9CMfGaGtrnj8SPbt04mQDhbiQ">
```

Comment:
These hidden fields have nothing to do with our app and should be removed.

Reply:
Fixed: Removed hidden fields.

Reply:
There is Drupal-specific markup here: all of the fieldsets, fieldset wrapper divs, legends and divs with webform related classes should not be in the code.

Reply:
@cdoyle-temple Drupal tags removed.

Also removed redundant labels.